source/_integrations/vodafone_station.markdown

```diff
@@ -63,6 +63,52 @@ There is support for the following platform types within Home Assistant:
 - **Sensor** - external IP address, uptime, firmware, resources and network monitors.
 - **Button** - restart router, dsl/fiber/internet key connections.
 
+## Examples
+
+### Automation: reconnect / get new IP every night
+
+```yaml
+automation:
+- alias: "Reconnect Vodafone Station (Fiber)"
+  triggers:
+    - trigger: time
+      at: "05:00:00"
+  actions:
+    - action: button.press
+      target:
+        entity_id: button.vodafone_station_xxxx_reconnect_fiber
+```
+
+### Automation: notify connected device not home
+
+```yaml
+automation:
+- alias: "Apple TV disconnect"
+  triggers:
+    - platform: state
+      entity_id: device_tracker.appletv
+      to: "not_home"
+  actions:
+    -  action: notify.mobile_app_phone
+       data:
+         message: "TV lost network connection"
+```
```

Comment:
Does this have anything to do with the integration in some way?

Reply:
I think so as the integration provides device trackers for all connected devices, and this is one of the main used feature.